js/functions.js

```diff
@@ -0,0 +1,77 @@
+//'Палиндром';
+
+function checkPalindrome(string) {
+  const newString = string.toLowerCase().replaceAll(' ', '').split('').reverse().join('');
+  string = string.toLowerCase().replaceAll(' ', '');
+  if(string === newString) {
+    return 'isPalindrome';
+  } else {
+    return 'isNotPalindrome';
+  }
```

Comment:
вообще по заданию нужно возвращать true или false  ), но что бы покороче было , можешь сразу возвращать результат сравнения string и newString, типа 
```suggestion
  return string === newString;
```

Reply:
Очень коротко и лаконично, переделал спасибо